pkg/blackduck/ctl_blackduck.go

```diff
@@ -136,6 +136,13 @@ func isValidSize(size string) bool {
 
 // CheckSpecFlags returns an error if a user input was invalid
 func (ctl *Ctl) CheckSpecFlags() error {
+	// log.Printf("ctl.Start: %v \n", ctl.Start)
+	// log.Printf("ctl.Stop: %v \n", ctl.Stop)
+	// log.Printf("ctl.Spec.DesiredState: %v \n", ctl.Spec.DesiredState)
+	if true == ctl.Start && true == ctl.Stop {
+		return fmt.Errorf("Both start and stop cannot be same value")
```

Comment:
This message is a bit confusing.  Suggesting something like "invalid options.  Only one of start or stop is allowed"